pylint_odoo/checkers/no_modules.py

```diff
@@ -655,9 +655,10 @@ def camelize(self, string):
         return re.sub(r"(?:^|_)(.)", lambda m: m.group(1).upper(), string)
 
     def formatversion(self, string):
-        self.config.manifest_version_format_parsed = \
-            self.config.manifest_version_format % dict(
-                valid_odoo_versions='|'.join(self.config.valid_odoo_versions))
+        self.config.manifest_version_format_parsed = (
+            self.config.manifest_version_format.format(
+                valid_odoo_versions='|'.join(
+                    map(re.escape, self.config.valid_odoo_versions))))
```

Comment:
Can we use some intermediary variables here? I'm having a bit of trouble following all the nested method calls IMO